src/styles/blocks/card.scss

```diff
@@ -0,0 +1,143 @@
+
+/* #region main */
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-self: center;
+  padding: 0 40px;
+  margin: 0 auto;
+  max-width: 1024px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  color: $text-secondary-color;
+  margin-top: 50px;
+  margin-right: 48px;
+  padding-left: 16px;
+  padding-right: 16px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__photo {
+    display: inherit;
+    padding-top: 32px;
+    padding-bottom: 40px;
+  }
+
+  &__photo-img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__description {
+    width: 166px;
+    height: 40px;
+    padding-bottom: 4px;
+
+    &-text {
+      font-family: inherit;
+      font-weight: 500;
+      font-style: normal;
+      font-size: 12px;
+      line-height: 18px;
+      color: $text-accent-color;
+    }
+  }
+  &__product-code {
+    align-self: flex-start;
+    line-height: 13px;
+    padding-bottom: 16px;
+
+    &-text {
+      font-family: inherit;
+      font-size: 10px;
+      font-weight: 400;
+      font-style: normal;
+      color: $text-secondary-color;
+    }
+  }
+  &__review {
+    width: 166px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: flex-end;
+    padding-bottom: 24px;
+
+    &-text {
+      font-family: inherit;
+      font-size: 10px;
+      line-height: 14px;
+      text-align: right;
+      color: $text-accent-color;
+    }
+  }
+  &__price {
+    width: 166px;
+    line-height: 18px;
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+    padding-bottom: 17px;
+
+    &-text {
+      display: flex;
+      align-self: flex-end;
+      flex-grow: 1;
+      font-family: inherit;
+      font-size: 12px;
+    }
+
+    &-number {
+      font-family: inherit;
+      font-size: 16px;
+      font-weight: bold;
+      color: $text-accent-color;
+    }
+  }
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    background-color: $main-color;
+    border: 1px solid $main-color;
+    border-radius: 5px;
+    margin-bottom: 15px;
+
+    &-link {
+      text-align: center;
+      font-family: inherit;
+      font-weight: 700;
+      font-size: 14px;
+      line-height: 16px;
+      text-transform: uppercase;
+      color: #fff;
+      text-decoration: none;
+    }
+
+    &:hover {
+      background-color: #fff;
+      color: $main-color;
+      cursor: pointer;
```

Comment:
cursor: pointer should be given to &-link class, not only on hover

Reply:
done